graphistry/feature_utils.py

```diff
@@ -1959,7 +2108,8 @@ def _featurize_nodes(
         X_resolved = resolve_X(ndf, X)
         y_resolved = resolve_y(ndf, y)
 
-        feature_engine = resolve_feature_engine(feature_engine)
+        res.feature_engine = feature_engine
```

Comment:
keep as a local variable, otherwise need to add to the class interface: https://github.com/graphistry/pygraphistry/blob/master/graphistry/Plottable.py